lib/openxml/docx/style.rb

```diff
@@ -84,11 +84,19 @@ def install_table_properties
       end
 
       def property_xml(xml)
+        style_property_xml(xml)
+
         return table.property_xml(xml) if table_style?
         character.property_xml(xml)
         paragraph.property_xml(xml) if paragraph_style?
       end
 
+      def style_property_xml(xml)
+        props = properties.keys.map(&method(:send)).compact
```

Comment:
Nice :) This is a very Rubyish line: chaining `Enumerable` methods, `&method(:send)`. :+1:

I'm _slightly_ worried about `&method(:send)` and **only** because it's `:send` and because ``:` `` is a method on every object (try it: ``"a".send :`, "cat /etc/passwd"``).

Could an attacker put something malicious into `properties`?

Is there a way to get a property from a key other than `send`? Like, does this work?

```ruby
props = properties.keys.map { |property| self[property] }.compact
```